### PR TITLE
(chore): environment to build Jekyll locally (via Docker)

### PR DESCRIPTION
**Pull request by linuxwolf**
Necessary files in order to build/serve GitHub-Pages styled Jekyll site locally.  A minimal README for using the dockerized environment is included.

### CODE REVIEW

**Comment by linuxwolf** [comment]
@devinreams README updated to be more complete.  LMK if that works.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+git clone https://github.com/mozilla-lockbox/mozilla-lockbox.github.io
+cd mozilla-lockbox.github.io
+docker build --tag mozilla-lockbox.github.io .
+```
+
+Then create and run a container to run a local Jekyll server:
+
+```bash
+cd mozilla-lockbox.github.io
+docker run --rm --tty --interactive --publish 4000:4000 --volume $PWD:/srv/jekyll mozilla-lockbox.github.io
```
Comment (devinreams) [inline README.md]: @linuxwolf Sorry if this is a "just on my machine" problem and I did something to my environment, but I've searched high and low and can't tell why I'm getting this error when I run this `docker run` command:

>su-exec: /usr/gem/bin/bundle: No such file or directory

I have no idea why _anything_ would expect `/usr/gem` to exist (it doesn't locally here). I feel like its related to this but can't tell: https://github.com/jekyll/docker/blob/master/repos/jekyll/Dockerfile

Comment (linuxwolf) [reply README.md]: \-:

Something is screwy with the base image, but apparently not for me.

Comment (linuxwolf) [reply README.md]: Looks like something with that base image broke between yesterday and today.  /sigh

**Comment by devinreams** [comment]
@linuxwolf from our quick chat yesterday, I'm still unable to get Docker running as expected:

screen shot 2017-11-08 at 11 36 19 am

Though, I _am_ able to get the site building locally myself:

```
bundle install
bundle exec jekyll serve
```

From [GitHub Help](https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/).

**Comment by linuxwolf** [comment]
@devinreams you might need to take steps to update the images, or blow everything away and start over (e.g., `docker rmi jekyll/jekyll`)

**Pull request merged**